Bump version and update changelog (#1176)
svix/svix-webhooks: ChangeLog.md

## Next
* 

## Version 1.16.0

> * Add download badges to README.

Reviewer: ```suggestion
```
Our customers don't care about this. The Changelog should be a summary that only includes the things our customers care about.

> * Server: Add 'tag' parameter to list-message for go,js,python.

Reviewer: ```suggestion
* Server: Add `tag` parameter to list-message for Go, JavaScript, and Python.
```
This is customer facing so should be nicely formatted.

* Server: backport expired message cleaner updates.
* Server: expired message cleaner cleanup.

> * Libs: fix descriptions in package repositories.

Reviewer: ```suggestion
```
